lib/screens/login/login2.dart

```diff
@@ -19,6 +23,16 @@ class _LoginFirebaseScreenState extends State<LoginFirebaseScreen> {
   final scaffoldKey = new GlobalKey<ScaffoldState>();
   String _email, _password;
 
+  @override
+  void initState() {
+    new Future.delayed(Duration.zero, () {
```

Comment:
Let's not use `new` as it's optional and may confuse people. Feel free to remove it if you find it in other places.